cmd/humioctl/util.go

```diff
@@ -137,3 +139,22 @@ func printDetailsTable(cmd *cobra.Command, data [][]format.Value) {
 func printOverviewTable(cmd *cobra.Command, header []string, data [][]format.Value) {
 	format.FormatterFromCommand(cmd).Table(header, data)
 }
+
+func getBytesFromFile(filePath string) ([]byte, error) {
+	// #nosec G304
+	return os.ReadFile(filePath)
+}
```

Comment:
I think commands ought to support `stdin` (and stdout for export) in general.

Suggestion:

```suggestion
func getBytesFromFile(filePath string) ([]byte, error) {
  if filePath == "-" {
    return io.ReadAll(os.Stdin)
  }
	// #nosec G304
	return os.ReadFile(filePath)
}
```

Or pass `*cobra.Command` and use `(*cobra.Command).InOrStdin()`